examples/experimental/imagenet_gaea/gcp_bucket_helper.py

```diff
@@ -0,0 +1,45 @@
+import os
+import time
+import random
+
+# Helper functions for GCP from https://cloud.google.com/storage/docs/listing-objects#code-samples
+def list_blobs(storage_client, bucket_name, prefix=None):
+    """Lists all the blobs in the bucket."""
+    blobs = storage_client.list_blobs(bucket_name, prefix=prefix)
+    paths = []
+    for b in blobs:
+        paths.append(b.name)
+    return paths
+
+
+def download_blob(storage_client, bucket_name, source_blob_name, destination_file_name, n_retries=4, max_backoff=8):
```

Comment:
FYI we have https://github.com/determined-ai/determined/blob/e0dd3e51f33e441fe4b6a8aa7cb3591f204f2af1/harness/determined/util.py#L19 which looks fairly similar.

Reply:
Thanks for the pointer!  I wound up implementing my own because I wanted to download to the filesystem directly.